### PR TITLE
Add mock

### PR DESCRIPTION
**Pull request by samolabams**

### CODE REVIEW

**Comment by adesokanayo** [comment]
Here is  my comment

File: Dangerfile [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
+# # Don't let testing shortcuts get into master by accident
+# fail("fdescribe left in tests") if `grep -r fdescribe specs/ `.length > 1
+# fail("fit left in tests") if `grep -r fit specs/ `.length > 1
+
+newNoEncryptOptions = 0
+git.diff.each do |chunk|
+    if chunk.path.match(/src\/el/)
+      chunk.patch.lines.grep(/^+/).each do |added_line|
+          if added_line.match(/NoEncrypt:\s*true/)
+            newNoEncryptOptions = newNoEncryptOptions + 1
```
Comment (azeezadio) [inline Dangerfile:22]: the occurrences of newNoEncryptOptions should also reset itself at the end

Comment (samolabams) [reply Dangerfile:22]: > the occurrences of newNoEncryptOptions should also reset itself at the end

This will be added

File: src/el/main.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package main
```
Comment (adesokanayo) [inline src/el/main.go:1]: Why do we have a main.go file here anyways , do we want to take it out

**Review by adesokanayo — CHANGES_REQUESTED** [review]

File: src/el/main.go (hunk 2 above)
Comment (samolabams) [reply src/el/main.go:1]: I will

File: src/el/redis/redis.go [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import "fmt"
+
+type cacheOptions struct {
+	NoEncrypt bool
+}
+
+func ConnectToRedis() error {
+	options := cacheOptions{NoEncrypt: true}
+
+	fmt.Println("connecting:", options)
```
Comment (adesokanayo) [inline src/el/redis/redis.go:12]:
```suggestion
	fmt.Println("connecting to redis with :", options)
```

**Review by adesokanayo — CHANGES_REQUESTED** [review]

File: Dangerfile [hunk 4]
```diff
@@ -1,13 +1,40 @@
...
+          end
+      end
+      chunk.patch.lines.grep(/^-/).each do |removed_line|
+          if removed_line.match(/NoEncrypt:\s*true/)
+            newNoEncryptOptions = newNoEncryptOptions - 1
+          end
+      end
+    end
+end
```
Comment (adesokanayo) [inline Dangerfile:31]: look great.

**Review by adesokanayo — APPROVED** [review]
ok

**Pull request closed without merging**